egs/wsj/s5/steps/online/nnet2/train_diag_ubm.sh

```diff
@@ -89,6 +89,15 @@ for f in $data/feats.scp "$online_cmvn_config" $srcdir/splice_opts $srcdir/final
    [ ! -f "$f" ] && echo "$0: expecting file $f to exist" && exit 1
 done
 
+if [ -d "$dir" ]; then
+  bak_dir=$(mktemp -d ${dir}/backup.XXX);
+  echo "$0: Directory $dir already exists. Backing up models in ${bak_dir}";
```

Comment:
This is actually the script for training the diagonal UBM.  So I can just refer to it as such here, and say "i-vector extractor" in the next script.